### ISSUE TITLE
{FEATURE} Range Slider Packages Skeleton

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/geoview#1379]
Create the Range Slider package skeleton and proof of concept

- [ ] Create the package
- [ ] Create the basic container whit the slider control
- [ ] Listen to layers with temporal dimension, if one, take the first one and build the slider 
- [ ] Slider filter the map

Once the proof of concept is done, we will create new issue for all the functionalities

**Comment by jolevesq** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/GeoView-Web-Team-6332f4560a41153d1fefbcbc/issues/Canadian-Geospatial-Platform/geoview/1379?planning-poker) with Zenhub @DamonU2
